fog.gemspec

```diff
@@ -45,7 +45,7 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = %w[README.md]
 
   ## Ruby version
-  s.required_ruby_version = ">= 1.9.3"
+  s.required_ruby_version = ">= 2"
```

Comment:
I think we should probably leave this as 1.9.3 for now, though dropping it also in time would be good.